websites/G/GenoAnime/presence.ts

```diff
@@ -0,0 +1,145 @@
+          duration = data.iframe_video.duration;
+          paused = data.iframe_video.paused;
+      }
+  }
+);
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+          largeImageKey: "genoanime",
+          startTimestamp: browsingStamp
+      },
+  title = document.title.slice(0, -13); //title of the page
+  if (document.location.pathname == "/") {
+      presenceData.details = "Exploring Genoanime";
+  } else if (document.location.pathname.includes("/browse")) {
+      presenceData.details = "Exploring Genoanime library";
+      presenceData.buttons = [{
+          label: "View Library",
+          url: document.location.href
+      }];
+  } else if (document.location.pathname.includes("/details")) {
+      presenceData.details = `Checking Synopsis`;
+      presenceData.state = document.querySelector(
+          ".anime__details__title h3"
+      ).textContent;
+      anime_breadcumb = String(
+          document.querySelector(
+              "#container > section > div > div.anime__details__content > div > div.col-lg-9 > div > div.anime__details__btn > a.watch-btn"
+          )
+      );
+      presenceData.buttons = [{
+              label: "Watch It",
+              url: String(anime_breadcumb)
+          },
+          {
+              label: "Check Synopsis",
+              url: document.location.href
+          }
+      ];
+  } else if (document.location.pathname.includes("/watch")) {
+      presenceData.details = title;
+      presenceData.state =
+          "Episode " + String(document.location.href.split("episode=")[1]);
+      anime_breadcumb = String(document.querySelector("#anime_details_breadcrumbs"));
+      presenceData.buttons = [{
+              label: "Watch Episode",
+              url: document.location.href
+          },
+          {
+              label: "Check Synopsis",
+              url: String(anime_breadcumb)
+          }
+      ];
+      if (iFrameVideo) {
+          timestamps = presence.getTimestamps(
+              Math.floor(currentTime),
+              Math.floor(duration)
+          );
+      } else {
+          video = document.querySelector("div > div.plyr__video-wrapper > video");
+          if (video) {
+              currentTime = video.currentTime,
+              duration = video.duration,
+              paused = video.paused,
+              timestamps = presence.getTimestamps(
+                  Math.floor(currentTime),
+                  Math.floor(duration)
+              );
+          }
+      }
+
+      if (!isNaN(duration)) {
+          presenceData.smallImageKey = paused ? "pause" : "play";
+          presenceData.smallImageText = paused ?
+              (await strings).pause :
+              (await strings).play;
+          presenceData.startTimestamp = timestamps[0];
+          presenceData.endTimestamp = timestamps[1];
+          if (paused) {
+              delete presenceData.startTimestamp;
+              delete presenceData.endTimestamp;
+          }
```

Comment:
```suggestion
          presenceData.smallImageKey = paused ? "pause" : "play";
          presenceData.smallImageText = paused ?
              (await strings).pause :
              (await strings).play;
          presenceData.endTimestamp = timestamps[1];
          if (paused) {
              delete presenceData.endTimestamp;
          }
```
don't need startTimestamp.